### PR TITLE
Added support for Dynamic Diagrams

### LINKED ISSUE #234: Dynamic Diagrams are rendered, but not displayed
Hey everyone and thank you for building this awesome tool!
I am currently evaluating whether we want to use it in combination with Gitlab Pages for our engineers. It looks very promising!

But I am wondering, whether [Dynamic Diagrams](https://c4model.com/#DynamicDiagram) are supported: The diagrams are rendered (as puml, svg and png, everything seems fine), but I can't find them on the website. Are they displayed anywhere or does this feature not (yet) exist?

The [example](https://structurizr.com/dsl?example=big-bank-plc) contains such a diagram called "SignIn".

Thanks and best regards,
Marius

### PR DESCRIPTION
**Pull request by qtzar**
Dynamic Diagrams are now exposed as a tab on the software system.
closes #234

### CODE REVIEW

File: src/test/kotlin/nl/avisi/structurizr/site/generatr/site/model/SoftwareSystemPageViewModelTest.kt [hunk 1]
```diff
@@ -146,6 +148,18 @@ class SoftwareSystemPageViewModelTest : ViewModelTest() {
         assertThat(getTab(viewModel, Tab.COMPONENT).visible).isTrue()
     }
 
+    @Test
+    fun `dynamic views tab only visible when component diagrams available`() {
```
Comment (qtzar) [inline src/test/kotlin/nl/avisi/structurizr/site/generatr/site/model/SoftwareSystemPageViewModelTest.kt:152]:
```suggestion
    fun `dynamic views tab only visible when dynamic diagrams available`() {
```

File: src/main/kotlin/nl/avisi/structurizr/site/generatr/site/model/SoftwareSystemDynamicPageViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package nl.avisi.structurizr.site.generatr.site.model
+
+import com.structurizr.model.SoftwareSystem
+import nl.avisi.structurizr.site.generatr.hasComponentViews
```
Comment (jp7677) [inline src/main/kotlin/nl/avisi/structurizr/site/generatr/site/model/SoftwareSystemDynamicPageViewModel.kt:4]: This import is not needed.

**Review by jp7677 — CHANGES_REQUESTED** [review]
Just one nit, otherwise LGTM!

**Review by jp7677 — APPROVED** [review]

**Comment by mwz** [comment]
Hi, any chance for this to be released anytime soon?

**Pull request merged**